WearSpeakerSample/wear/src/main/java/com/example/android/wearable/speaker/ControlDashboard.kt

```diff
@@ -63,7 +63,7 @@ fun ControlDashboard(
         circle = circle,
         mic = mic,
         play = play,
-        music = music,
+        music = music
```

Comment:
allow trailing comma was set to true for this sample, so how come it stripped it?

Reply:
I can flip to false in .editorconfig so it's consistent. I'm about to go on holidays, so would like build green and passing and then forget about this :)

Or we can raise a bug, similar reports https://github.com/diffplug/spotless/issues/1193 and it's defined here and one more place https://github.com/pinterest/ktlint/blob/d64733ba1711c592fb78fe04d3472c5294004450/ktlint-ruleset-standard/src/main/kotlin/com/pinterest/ktlint/ruleset/standard/TrailingCommaOnCallSiteRule.kt#L290